openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1257,17 +1257,17 @@ HTTP/1.1 204 No Content
 
 ## Notification Error Response
 
-If the Notification Request does not contain an Access Token or contains an invalid Access Token, the Notification Endpoint returns an Authorization Error Response such as defined in section 3 of [@!RFC6750].
+If the Notification Request does not contain an Access Token or contains an invalid Access Token, the Notification Endpoint returns an Authorization Error Response, as defined in Section 3 of [@!RFC6750].
```

Comment:
the "such as" here was intentional so as not to preclude `WWW-Authenticate` challenges with schemes other than `Bearer` like `DPoP` 

```suggestion
If the Notification Request does not contain an Access Token or contains an invalid Access Token, the Notification Endpoint returns an Authorization Error Response, such as defined in Section 3 of [@!RFC6750].
```